src/cpp/session/resources/themes/compile-themes.R

```diff
@@ -653,6 +653,8 @@
          sprintf(".xtermInvertBgColor { background-color: %s; }", foreground),
          ".xtermBold { font-weight: bold; }",
          ".xtermUnderline { text-decoration: underline; }",
+         ".xtermHyperlink { cursor: pointer; }",
+         ".xtermHyperlink:hover { text-decoration: underline; }",
```

Comment:
Minor edge case; if a user had previously imported a theme into RStudio (`.tmtheme`), then upgraded to RStudio with this new feature, the saved `.rstheme` won't have these two lines. I think they'd have to delete the theme and reimport it to correct that. Don't think we need to do anything about that, just mentioning it.

Reply:
I think it's a minor issue, because the link already get added `xtermUnderline` and `xtermColor5` so they do stand out. 

The `:hover` bit could be removed I suppose, it's only useful if the `xtermUnderline` style was cancelled. 

I had to add the `cursor: pointer` because it would only be inherited from `<a>` when there is an `href=` attribute, but then the linked page opened in the same tab in addition to triggering the rpc.

Reply:
We could add `anchor.setAttribute("style", "cursor: pointer;");` to be on the safe side.